Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18.4 AS build
+FROM golang:1.19 AS build
```

Comment:
Should maybe use the entire semver here?

[1.19.0-alpine3.16](https://hub.docker.com/layers/library/golang/1.19.0-alpine3.16/images/sha256-70df3b8f9f099da7f60f0b32480015165e3d0b51bfacf9e255b59f3dd6bd2828?context=explore)

Reply:
Also, was wondering why Dependabot wasn't handling this... and it looks like it did?

https://github.com/nuance-mix/grpc-health-probe/pull/16/files

We can probably close this PR.

Reply:
Ah, yeah, looks like Dependabot is wrong... ok, approving.